Supporting custom vertex normals for polymeshes.

sirpalee opened the pull request:
**Changes proposed in this pull request**
- Converting normal primvars to nlist and nidxs
- Unifying normal and uv handling code
- Reading points during primvar iteration

**Issues fixed in this pull request**
Fixes #1

sebastienblor commented:
The commit nicely factorizes the code that was already done to convert uvs.
Isn't there a typo in the function name _ConvertFaceVaryingPrimvar**y**ToBuiltin ?

sirpalee commented:
> The commit nicely factorizes the code that was already done to convert uvs.
> Isn't there a typo in the function name _ConvertFaceVaryingPrimvar**y**ToBuiltin ?

Good catch, thanks! Updated the pull request and renamed the function.
